specification/e-referrals-service-api.yaml

```diff
@@ -131,6 +131,16 @@ info:
     This API is primarily [RESTful](https://digital.nhs.uk/developer/guides-and-documentation/api-technologies-at-nhs-digital#basic-rest) 
     and based on [FHIR](https://digital.nhs.uk/developer/guides-and-documentation/api-technologies-at-nhs-digital#fhir) global standards for health care data exchange.
 
+    This API supports FHIR STU3 and FHIR R4.
+
+    ### FHIR STU3
+
+    The e-RS FHIR STU3 is built against a set of profiles called [CareConnect](https://nhsconnect.github.io/CareConnectAPI/)
+
+    ### FHIR R4
+
+    The e-RS FHIR R4 API is built against [FHIR UK Core](https://digital.nhs.uk/services/fhir-uk-core). The canonical URL listed in the schema can be resolved with https://simplifier.net/resolve
```

Comment:
The e-RS FHIR R4 API is built against the [NHS Digital Profile](https://simplifier.net/guide/nhsdigital?version=current
), which is built on top of [FHIR UK Core](https://digital.nhs.uk/services/fhir-uk-core). The canonical URLs listed in the schema can be resolved with https://simplifier.net/resolve.